ivy/functional/ivy/creation.py

```diff
@@ -974,7 +977,82 @@ def eye(
 
     Both the description and the type hints above assumes an array input for simplicity,
     but this function is *nestable*, and therefore also accepts :class:`ivy.Container`
-    instances in place of any of the arguments.
+    instances in place instead of the arguments.
```

Comment:
This line should be left in its origenal form as it should mean that the datatype passed into any of the arguments can also be containers